src/xo-app/vm/tab-logs.js

```diff
@@ -53,34 +39,30 @@ const LOG_COLUMNS = [
   })
 })
 export default class TabLogs extends Component {
-  _deleteAllLogs = () => map(this.props.logs, deleteMessage)
-
   render () {
-    const { logs } = this.props
-
-    if (isEmpty(logs)) {
-      return <Row>
-        <Col className='text-xs-center'>
-          <br />
-          <h4>{_('noLogs')}</h4>
-        </Col>
-      </Row>
-    }
+    const INDIVIDUAL_ACTIONS = [
+      {
+        label: 'deleteLog',
+        icon: 'delete',
+        handler: deleteMessage
+      }
+    ]
+    const GROUPED_ACTIONS = [
+      {
+        label: 'deleteLogs',
+        icon: 'delete',
+        handler: logs => map(logs, deleteMessage)
+      }
+    ]
```

Comment:
Avoid creating objects in the render method as new objects will be created on each render.